services/socket.js

```diff
@@ -52,9 +52,6 @@ const activateSocket = (io) => {
     if (!timeInterval) {
       timeInterval = setInterval(() => countdown(socket.user), COUNTDOWN_INTERVAL_SECOND);
     };
-    socket.on('message', (message) => {
```

Comment:
keep this code and delete the below. `message` is an object so if you handled it by the below way,  `message` will display `null` (I've tested before comment)